ros/api/v1/recommendation_ratings.py

```diff
@@ -28,20 +28,28 @@ def post(self):
 
         data = json.loads(request.data)
         inventory_id = data['inventory_id']
-        rating = int(data['rating'])
-        if rating not in [c.value for c in RatingChoicesEnum]:
-            abort(
-                422,
-                message=(
-                    "{} is not a valid value for rating".format(rating)))
+        allowed_choices = [c.value for c in RatingChoicesEnum]
+
+        def invalid_value_msg(val):
+            return (
+                f"'{val}' is invalid value for rating."
+                f'Possible values - { *allowed_choices, }'
```

Comment:
Nitpick :

Let's have consistent surrounding quotes?

Reply:
@saltgen - In above sentence,  given value for rating is enclosed in single quotes (') so used ("") double quotes.
Did you mean to add double quotes on this line-36, correct?

Reply:
Yes, L36, correct